Library/ViewModels/PledgePaymentPlansViewModelTest.swift

```diff
@@ -71,9 +67,20 @@ final class PledgePaymentPlansViewModelTests: TestCase {
     withEnvironment {
       self.vm.inputs.viewDidLoad()
 
+      self.vm.inputs.configure(with: self.selectionData)
+      self.reloadPaymentPlansPlanType.assertValues([.pledgeInFull])
       self.vm.inputs.didSelectPlanType(.pledgeOverTime)
-      self.reloadPaymentPlansPlanType.assertValues([.pledgeOverTime])
+      self.reloadPaymentPlansPlanType.assertValues([.pledgeInFull, .pledgeOverTime])
       self.notifyDelegatePaymentPlanSelected.assertValues([.pledgeOverTime])
     }
   }
+
+  func testPaymenPlans_TermsOfUseTapped() {
+    self.vm.inputs.viewDidLoad()
+
+    self.vm.inputs.configure(with: self.selectionData)
+    self.vm.inputs.didTapTermsOfUse(with: .terms)
```

Comment:
@scottkicks I checked the PLOT web version and they are showing the current Terms of use web page.